diff --git a/node_modules/google-caja-sanitizer/sanitizer.js b/node_modules/google-caja-sanitizer/sanitizer.js
index bf8ff98..43e8173 100644
--- a/node_modules/google-caja-sanitizer/sanitizer.js
+++ b/node_modules/google-caja-sanitizer/sanitizer.js
@@ -4919,16 +4919,16 @@ var html = (function(html4) {
 })(html4);


-exports.escapeAttrib = html.escapeAttrib;
-exports.makeHtmlSanitizer = html.makeHtmlSanitizer;
-exports.makeSaxParser = html.makeSaxParser;
-exports.makeTagPolicy = html.makeTagPolicy;
-exports.normalizeRCData = html.normalizeRCData;
-exports.sanitize = html.sanitize;
-exports.smartSanitize = function (string, urlX, idX) {
-  string = string.replace(/<([a-zA-Z]+)([^>]*)\/>/g, '<$1$2></$1>');
-  return html.sanitize(string, urlX, idX);
-}
-exports.sanitizeAttribs = html.sanitizeAttribs;
-exports.sanitizeWithPolicy = html.sanitizeWithPolicy;
-exports.unescapeEntities = html.unescapeEntities;
+// exports.escapeAttrib = html.escapeAttrib;
+// exports.makeHtmlSanitizer = html.makeHtmlSanitizer;
+// exports.makeSaxParser = html.makeSaxParser;
+// exports.makeTagPolicy = html.makeTagPolicy;
+// exports.normalizeRCData = html.normalizeRCData;
+// exports.sanitize = html.sanitize;
+// exports.smartSanitize = function (string, urlX, idX) {
+//   string = string.replace(/<([a-zA-Z]+)([^>]*)\/>/g, '<$1$2></$1>');
+//   return html.sanitize(string, urlX, idX);
+// }
+// exports.sanitizeAttribs = html.sanitizeAttribs;
+// exports.sanitizeWithPolicy = html.sanitizeWithPolicy;
+// exports.unescapeEntities = html.unescapeEntities;
